eth/executionclient/execution_client.go

```diff
@@ -172,11 +163,13 @@ func (ec *ExecutionClient) fetchLogsInBatches(ctx context.Context, startBlock, e
 			}
 
 			start := time.Now()
-			results, err := ec.client.FilterLogs(ctx, ethereum.FilterQuery{
+			query := ethereum.FilterQuery{
 				Addresses: []ethcommon.Address{ec.contractAddress},
 				FromBlock: new(big.Int).SetUint64(fromBlock),
 				ToBlock:   new(big.Int).SetUint64(toBlock),
-			})
+			}
+
+			results, err := ec.subdivideLogFetch(ctx, query)
 			if err != nil {
 				ec.logger.Error(elResponseErrMsg,
```

Comment:
we should move the `elResponseErrMsg` logging into `subdivideLogFetch()`.
`elResponseErrMsg` specifically refers to `eth_getLogs`, but `subdivideLogFetch()` can now fail for other reasons too (e.g. context cancellation, client closed, internal errors).
Logging it outside assumes the error is always from `eth_getLogs`, which may no longer be true.